src/app/tide/spectrum_preprocess2.cc

```diff
@@ -222,7 +222,7 @@ void ObservedPeakSet::PreprocessSpectrum(const Spectrum& spectrum, int charge,
       if (highest_intensity == 0) {
         continue;
       }
-      normalizer = 50.0 / highest_intensity;
+      normalizer = 25000000.0 / highest_intensity;
```

Comment:
Why did this value change? The original paper used a value of 50.

Reply:
In the original implementation first the experimental peaks are normalized to the range between 0 and 50. Later in the Tide-Search, the double-valued experimental spectrum vector ```peaks_``` is integerized and the experimental peak intensities are multiplied by a large integer: 500000. I have combined these two steps into one calculation. hence 50*500000=25000000.0

Reply:
Can you add the above as a comment in the code, to make clear where this number comes from?

Reply:
Sure thing! Done.